package.json.orig

```diff
@@ -0,0 +1,79 @@
+{
```

Comment:
What is the purpose of this file?

Reply:
good catch - this should be removed - it gets created automatically during a merge and shouldn't have been included in the commit